deploy-board/deploy_board/templates/environs/env_landing.html

```diff
@@ -372,6 +372,9 @@ <h4 class="panel-title pull-left pointer-cursor">
                 </div>
                 {% endif %}
             {% endif %}
+            <h6 class="stage-type-badge">
+                {{env.stageType|default:"NO STAGE TYPE"}}
```

Comment:
added NO STAGE TYPE as stageType is an optional field from the API according to the docs

Reply:
I'm thinking we could show a warning if the stage type is missing, to encourage users to set it. The warning could be similar to the one in the screenshot.

However, I think we could discuss this separately, so this change looks good to me :+1: